contents/handbook/growth/sales/expansion-and-retention.md

```diff
@@ -42,23 +42,38 @@ You'll also go through the same [contracting process](/handbook/growth/sales/con
 AEs also do expansion at PostHog. This is because we are constantly on a sales footing with customers - for the most part, we don't do steady state account management with an arbitrary 10% uplift at renewal team. There are two primary ways we look to drive expansion with a customer:
 
 **Cross sell new products**
-- First you need to find out who cares about the new product - is it the existing team or the new team?
-- Then you need to find out what are they using now for other tools - surface this during the check in calls that you already have scheduled as part of onboarding.
-- At this point
-  - If it's already a mature product we have shipped, you should aim to show how the product complements what they _already_ are using in PostHog - don't just arbitrarily sell in a product for the sake of it. 
-  - If it's something in beta or coming soon, you should start giving them sneak peeks of what's on our [roadmap](/roadmap). You can also schedule a feedback session with the relevant product engineer if they’re a great fit - customers _love_ this. Again, consider playing the founder card for something _really_ new and big. 
+
+- First you need to find out who cares about the problem that our other products solve - is it the existing team or the new team?
+  - Use a tool like [The Org](https://theorg.com/) or [Chartloop](https://www.chartloop.com/#) to help you identify new people. 
+- Then you need to find out what are they using now to solve the problem (if anything) - surface this during the check in calls that you already have scheduled as part of onboarding if it's the existing team. If you're talking a new team, run this as a [new sales process](/handbook/growth/sales/new-sales). 
+- Your approach will depend on the product that makes sense here:
+  - If it's already a mature product we have shipped, you should aim to show how the product complements what they _already_ are using in PostHog - don't just arbitrarily sell in a product for the sake of it. For example, you can say ‘other customers that look like you are doing X, this is what we’re seeing’.
+  - If it's something in beta or coming soon, you should start giving them sneak peeks of what's on our [roadmap](/roadmap). You can also schedule a feedback session with the relevant product engineer if they’re a great fit - customers _love_ this. Again, consider playing the founder card for something _really_ new and big.
+- Understanding the blockers to using other products - these could be:
+  - Privacy/compliance concerns (e.g. viewing session recordings) - we have a lot of documentation on this
+  - Already doing it in house/with something else - demonstrated cool ways in which the products integrate and save their team time
+  - May be too far down the line with their own data warehouse - this is hard to overcome at this stage
```

Comment:
It's hard to replace an existing data warehouse if they are at this stage but we can instead position it as "enrich your PostHog data with what's in your current warehouse"

Reply:
Added